miden-tx/src/host/tx_authenticator.rs

```diff
@@ -166,8 +166,8 @@ impl TransactionAuthenticator for () {
 
 #[cfg(test)]
 mod test {
+    use miden_lib::utils::{Deserializable, Serializable};
```

Comment:
on `bench-tx/src/main.rs` we had a `miden_lib::utils::Serializable` which was changed for `miden_tx::utils::Serializable`. Should we use the same one?

Reply:
I don't think it's necessary as they are the same